src/Runtime/OMTensorList.inc

```diff
@@ -35,9 +35,7 @@ struct OMTensorList {
    * Create an OMTensorList with specified OMTensor pointer array
    * and the size of the array
    */
-  OMTensorList(OMTensor *omts[], int64_t n) : _omts(omts), _size(n) {
-    _owning = false;
-  };
+  OMTensorList(OMTensor *omts[], int64_t n) : _omts(omts), _size(n){};
```

Comment:
Constructor needs to use `new` to make a copy of `omts`. Destructor uses `delete` to free the copy.

Reply:
Thanks for this feedback... will update.

Reply:
Constructor could explicitly call one of the explicit constructors... to avoid replicated code.

Reply:
I am fine with either way... do you agree @gongsu832 or do you feel otherwise?

Reply:
A more general constructor calls a more specific constructor is a pretty common practice.

Reply:
Okay done!